org.arrl.trustedqsl.json

```diff
@@ -0,0 +1,158 @@
+		"/share/man/man5/*",
+		"/share/man/man7/*",
+		"/share/man/man1",
+		"/share/man/man3",
+		"/share/man/man5",
+		"/share/man/man7",
+		"/include/openssl/*",
+		"/include/openssl",
+		"/lib64/pkgconfig/*",
+		"/lib64/pkgconfig"
+	    ]
+        },
+        {
+            "name" : "lmdb",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "cd libraries/liblmdb/ && make prefix=/app install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://github.com/LMDB/lmdb/archive/LMDB_0.9.23.tar.gz",
+                    "sha256" : "abf42e91f046787ed642d9eb21812a5c473f3ba5854124484d16eadbe0aa9c81"
+                }
+            ],
+	    "cleanup" : [
+		"/include/lmdb.h",
+		"/lib/liblmdb.a",
+		"/share/man/man1/*",
+		"/share/man/man1",
+		"/bin/mdb_copy",
+		"/bin/mdb_dump",
+		"/bin/mdb_load",
+		"/bin/mdb_stat"
+	    ]
+        },
+        {
+            "name" : "curl",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "./configure prefix=/app --with-openssl && make install"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://curl.se/download/curl-8.0.1.tar.gz",
+                    "sha256" : "5fd29000a4089934f121eff456101f0a5d09e2a3e89da1d714adf06c4be887cb"
+                }
+            ],
+	    "cleanup" : [
+		"/bin/curl",
+		"/include/curl/*",
+		"/include/curl",
+		"/lib/libcurl.a",
+		"/bin/curl-config",
+		"/share/man/man1/*",
+		"/share/man/man3/*",
+		"/share/man/man7/*",
+		"/share/man/man1",
+		"/share/man/man3",
+		"/share/man/man7",
+		"/share/aclocal/libcurl.m4",
+		"/share/aclocal",
+		"/lib/pkgconfig/libcurl.pc",
+		"/lib/pkgconfig"
+	    ]
+        },
+	{
+	    "name" : "wxWidgets",
+	    "buildsystem" : "cmake",
+    	    "config-opts" : [
+		    "-DwxUSE_OPENGL=No",
+		    "-DwxUSE_HTML=Yes",
+		    "-DwxUSE_STREAMS=Yes",
+		    "-DwxUSE_WXHTML_HELP=Yes",
+		    "-DCMAKE_BUILD_TYPE=Release",
+		    "-DwxUSE_UNICODE=Yes"
+	    ],
+	    "sources" : [
+		{
+		    "type" : "archive",
+        	    "url" : "https://github.com/wxWidgets/wxWidgets/releases/download/v3.2.2.1/wxWidgets-3.2.2.1.tar.bz2",
+        	    "sha256" : "dffcb6be71296fff4b7f8840eb1b510178f57aa2eb236b20da41182009242c02"
+	  	}
+	    ],
+	    "cleanup" : [
+			"/lib/wx",
+			"/bin/wx-config",
+			"/bin/wxrc",
+			"/bin/wxrc-3.2",
+			"/share/bakefile",
+			"/include/wx-3.2/wx/*",
+			"/include/wx-3.2/wx",
+			"/include/wx-3.2",
+			"/lib/cmake/wxWidgets/*",
+			"/lib/cmake/wxWidgets",
+			"/lib/cmmake"
+	    ]
+	},
+	{
+	    "name" : "tqsl",
+	    "buildsystem" : "cmake",
+    	    "config-opts" : [
+	        "-DTQSL_FLATPAK=Yes",
+		"-DCMAKE_BUILD_TYPE=Release"
+	    ],
+	    "sources" : [
+		{
+		    "type" : "archive",
+        	    "url" : "https://www.rickmurphy.net/lotw/tqsl-2.7.tar.gz",
```

Comment:
is this a fork? This is not the same as found at https://www.arrl.org/tqsl/tqsl-2.6.5.tar.gz

Reply:
This is the current head of this package. I'll get pulled to arrl.org when it's released.